test/distributed_stdlib_detection.jl

```diff
@@ -0,0 +1,22 @@
+@testset "Distributed.jl detection" begin
+    # Just loading Distributed should do nothing
+    cmd = `$test_exename $test_exeflags -e 'using Distributed, DistributedNext; @assert !DistributedNext._check_distributed_active()'`
+    stderr_buf = IOBuffer()
+    run(pipeline(cmd; stderr=stderr_buf))
+    stderr_str = String(take!(stderr_buf))
+    @test isempty(stderr_str)
+
+    # Only one of the two being active should also do nothing
+    cmd = `$test_exename $test_exeflags -e 'using Distributed, DistributedNext; Distributed.init_multi(); @assert !DistributedNext._check_distributed_active()'`
```

Comment:
Should we also test the other combination (`DistributedNext.init_multi()` only)?

Reply:
Yeah we might as well, added a test for it in fe0e79a (also took the liberty of cleaning up the test a bit).